docs/managed-victoriametrics/alertmanager-setup-for-deployment.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Alertmanager and VMAlert configuration for Deployment
+aliases:
+- /managed-victoriametrics/alertmanager-configuration.html
+---
+
+## Alerting stack configuration and Managed VictoriaMetrics
+
+ Managed VictoriaMetrics supports configuration for alerting rules and notifications for it with alertmanager.
+
+## Configure alertmanager
+
+ Managed VictoriaMetrics supports alertmanager with standard [configuration](https://prometheus.io/docs/alerting/latest/configuration/).
+Configuration menu located at `deployment` page under `Alertmanager` section.
+
+<img src="alertmanager_location.webp">
+ Configuration parameters have following limitations:
+
+### allowed receivers
+
+* `discord_configs`
+* `pagerduty_configs`
+* `slack_configs`
+* `webhook_configs`
+* `opsgenie_configs`
+* `wechat_configs`
+* `pushover_configs`
+* `victorops_configs`
+* `telegram_configs`
+* `webex_configs`
+* `msteams_configs`
+
+### forbidden keys
+
+ All configuration params with `_file` suffix is not allowed for security reasons.
```

Comment:
All configuration parameters with a _file suffix are not allowed for security reasons.